README.md

```diff
@@ -64,7 +74,7 @@ We also have official client-side implementations for
 [Java](https://github.com/metarhia/jstp-java)
 that work effortlessly on iOS and Android 🎉
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```

Comment:
Another option is to revert the centering of titles and remove three or so badges from the page completely.